From e1e51562a8a25892f8c6027b428ce06b0465ac0b Mon Sep 17 00:00:00 2001
From: Marek Marczykowski <marmarek@invisiblethingslab.com>
Date: Sat, 20 Oct 2018 11:16:05 +0200
Subject: [PATCH] anaconda: disable os prober
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

It tries to mount every existing block device, including VM images.

Signed-off-by: Frédéric Pierret <frederic.epitre@orange.fr>
---
 pyanaconda/bootloader.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/pyanaconda/bootloader.py b/pyanaconda/bootloader.py
index deab41cf1..9a4defb9f 100644
--- a/pyanaconda/bootloader.py
+++ b/pyanaconda/bootloader.py
@@ -1507,6 +1507,7 @@ class GRUB2(GRUB):
         defaults.write("GRUB_CMDLINE_XEN_DEFAULT=\"console=none dom0_mem=min:1024M dom0_mem=max:4096M iommu=no-igfx ucode=scan smt=off\"\n")
         defaults.write("GRUB_DISABLE_RECOVERY=\"true\"\n")
         defaults.write("GRUB_THEME=\"/boot/grub2/themes/system/theme.txt\"\n")
+        defaults.write("GRUB_DISABLE_OS_PROBER=\"true\"\n")
         defaults.close()
 
     def _encrypt_password(self):
-- 
2.14.4

